pep-0654.rst

```diff
@@ -1048,72 +1060,103 @@ clauses. We rejected this because it would be less obvious. The fact that we
 are handling ``ExceptionGroups`` of ``T`` rather than only naked ``Ts`` should be
 specified in the same place where we state ``T``.
 
-See Also
-========
 
-* An analysis of how exception groups will likely be used in asyncio
-  programs:
-  https://github.com/python/exceptiongroups/issues/3#issuecomment-716203284
+Programming Without except*
```

Comment:
```suggestion
Programming Without ``except *``
```
(Not sure if RsT likes code blocks in titles though.)

Reply:
It should handle them OK, but they will look awkward when rendered to HTML. So I personally try to avoid using them in titles.

Reply:
Maybe use single quotes then?

Reply:
Alright